### ISSUE TITLE
Bug: Keepalived crash if tunnel devices configured.

### ISSUE THREAD

**Issue by ywc689** [iqiyi/dpvs#702]
Version: DPVS 1.8.8, Keepalived: 2.0.19

Reproduce: Refer to the tutorial examples.
https://github.com/iqiyi/dpvs/blob/master/doc/tutorial.md#vdev-tun

**Comment by yifan1227** [comment]
in file keepalived/keepalived/check/check_parser.c, line 982:
```
static void
tunnel_handler(const vector_t *strvec)
{
	alloc_tunnel(vector_slot(strvec, 1));
}
```
add judgement:
```
if (!strvec)
    return;
```

**Comment by ywc689** [comment]
Fixed by pr #718
